accumulate.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudscale-ch/cloudscale-go-sdk/v2"
+	"os"
+	"time"
+)
+
+// Represents one data point ("fact") in the billing database. The actual value for the data point is not present in
+// this type, as this type is just a map key, and the corresponding value is stored as a map value.
+type AccumulateKey struct {
+	Query     string
+	Zone      string
+	Tenant    string
+	Namespace string
+	Start     time.Time
+}
+
+func (this AccumulateKey) String() string {
+	// This is the full "source" string as used by the appuio-cloud-reporting
+	return this.Query + ":" + this.Zone + ":" + this.Tenant + ":" + this.Namespace
+}
+
+/*
+This function gets all the bucket metrics from cloudscale and puts them into a map. The map key is the "AccumulateKey",
```

Comment:
(the one-line-per-sentence suggestion also applies here, if followed)
```suggestion
/*
accumulateBucketMetrics gets all the bucket metrics from cloudscale and puts them into a map. The map key is the "AccumulateKey",
```